core/controllers/domain_objects_validator.py

```diff
@@ -243,3 +243,49 @@ def validate_suggestion_images(files):
     # The files argument do not represent any domain class, hence dict form
     # of the data is returned from here.
     return files
+
+
+def validate_exploration_change(exp_change_dict):
+    """Validate the exploration change list
+
+    Args:
+        exp_change_dict: dict. The exploration change dictionary
+            that needs to be validated.
+
+    Returns:
+        exp_domain.ExplorationChange. The exploration change domain
+        object.
+    """
+    if exp_change_dict.get('cmd') is None:
+        raise base.BaseHandler.InvalidInputException(
+            'Missing cmd key in change dict')
+
+    exp_change_commands = [
+        command['name'] for command in
+        exp_domain.ExplorationChange.ALLOWED_COMMANDS
+    ]
+
+    if exp_change_dict['cmd'] in exp_change_commands:
+        exp_domain.ExplorationChange(exp_change_dict)
```

Comment:
What is the point of this statement if the result is not assigned to anything?

If this is just for validation purposes, a comment would be useful here.

Reply:
Yes you are correct it is for the validation. According to the new code we won't be requiring this part as we are returning the domain object that will automatically call the `validation` method. So I have removed the initialization part.
Thanks